lib/GaletteEvents/PluginGaletteEvents.php

```diff
@@ -144,7 +144,23 @@ public static function getPublicMenusItemsList(): array
      */
     public static function getDashboardsContents(): array
     {
-        return [];
+        /** @var Login $login */
+        global $login;
+
+        if ($login->isSuperAdmin()) {
+            return [];
+        }
+
```

Comment:
```suggestion
```

Calendar shows all events accessible to an user, not event for a particular one; therefore limitting access to super user seems useless (he has access from regular menu).

Reply:
All right ! Let's do that :smile: